libethereum/Client.cpp

```diff
@@ -132,17 +146,32 @@ Client::~Client() {
         cerror << "Instance of SkaleHost was not properly created.";
 
     m_signalled.notify_all();  // to wake up the thread from Client::doWork()
-    stopWorking();
 
     m_tq.HandleDestruction();  // l_sergiy: destroy transaction queue earlier
     m_bq.stop();               // l_sergiy: added to stop block queue processing
 
-    terminate();
+    m_bc.close();
+    std::cout << cc::success( "Blockchain is closed" ) << "\n";
```

Comment:
Better write to log - and not to stdout

Reply:
This can be removed after resolving problems with shutdown

Reply:
resolved